### DISCUSSION TITLE
Running code in a branch

### DISCUSSION THREAD

**Discussion by JELASALA** [community/community#57613] [Programming Help]
How do I test experimental code in a branch?  That is, when I use the URL for the index.html servers run the one in the main branch.  How do I invoke the index file in a different branch?

**Comment by DSDmark** [comment]
Hi @JELASALA,

Just create a new branch and test your code there. Here, how to create:- [**GitHub docs**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository).

And about the **index.html** file, I assume you means by deploying a site. So, you can change the branch or deploy in another branch like this:

- Go to your repo setting and open it, inside the setting you can see that:

![image](https://github.com/community/community/assets/85744816/00a4b6e1-c4c0-4cab-b855-f9948c9cd463)
